tests/test_api.py

```
@@ -33,34 +35,198 @@ def client():
+        "password": "SecurePass123",
+    }
+    resp = client.post(
+        "/api/auth/register",
+        data=json.dumps(payload),
+        content_type="application/json",
+    )
+    assert resp.status_code == 201, f"Fixture setup failed: {resp.get_json()}"
+    return payload  # return the raw payload so tests can reuse email/password
+
+
+def _get_jwt(client, email="test@example.com", password="SecurePass123") -> str:
```

Ruff (0.16.3)

[error] Possible hardcoded password assigned to function default: "password"

(S107)